docs/helm/upgrades.md

```diff
@@ -0,0 +1,99 @@
+# Upgrading the helm charts
+
+Our helm charts are continuously improved and sometimes arise the need to introduce a breaking change.
+
+To get an overview of the changes in each release, first take a look at the
+release notes that are available via [GitHub Releases](https://github.com/Alfresco/acs-deployment/releases).
+
+Here follows a more detailed explaination of any breaking change grouped by version in which they have been released.
+
+## To be released
```

Comment:
this implies that part of the release process should be to rename this section to the current release and make a new to be released section for the future (only for GA release or also Mx?)

Reply:
@gionn IMHO the value justifies the burden. I would personnally keep everything under `To be released` (so M & A releases as well), but I know that customers are using the charts -M releases.
Though that could be a first step of aligning this repo's master branch with what we do in Ansible. @vprovaggi any inputs? 

@anxumalo @Amy-Hyland We're thinking about introducing this to the helm documentation. I guess that's something of interest to you?

Reply:
I think it's reasonable to reference here also pre-release until we make the next GA. During the GA release process, we merge any -M* under the same GA version section.

Reply:
> @anxumalo @Amy-Hyland We're thinking about introducing this to the helm documentation. I guess that's something of interest to you?

@alxgomz Oh yes, it'll make it easier to follow what's changed when, as the docs are prepared for the GA release. Thanks for the heads up!! 👍🏼